webpack.config.js

```diff
@@ -170,6 +171,7 @@ module.exports = (env = {NODE_ENV: process.env.NODE_ENV || 'production'}) => ({
               WHISTLER_API_SERVICE_URL: JSON.stringify(
                 'https://whistler-prod.allnint.ciscospark.com/api/v1'
               ),
+              CALLING_SDK_VERSION: JSON.stringify(callingPackage.version),
```

Comment:
Added this in webpack, as I felt it that we can access it through the `process.env` variable across the codebase and hence easier to access the appropriate version number.
Open to any suggestions though.

Reply:
Just curious, why do we need to `JSON.stringify` a string?

Reply:
Thanks for the catch Kesava. Yes, I had added it under the assumption that we _might_ not be getting a string. There was also some older logic which I had used before and I forgot to remove this stringify. You are correct, it's not required. Will update the PR with the same changes and the VERSIOn string changes.